### ISSUE TITLE
Tutorial Doesn't Load Types

### ISSUE THREAD

**Issue by captainrandom** [btq-ag/keelung#13]
Following your tutorial: https://btq.gitbook.io/keelung/getting-started/hello-world

I've loaded the tutorial module, and the moment I try to define the echo function, I get the following error:
![Screenshot from 2023-03-12 13-32-45](https://user-images.githubusercontent.com/13500774/224572186-bafb1ea4-21df-44c9-8715-a9a0f939054f.png)

How do I resolve this?

**Comment by christam96** [comment]
Hey @captainrandom! 

The line `echo :: Comp Field` is a part of the `echo` function in `src/Tutorial.hs` and is not intended on being entered into the REPL. Rather, the `echo` function was loaded into the REPL when you did `:l Tutorial`. 

To proceed with the `Hello World` example, simply interpret the function in the REPL using `gf181 echo [] [3]`, which should echo `[3]`. Similarly, try running `gf181 echo [] [6]` to echo `[6]`.

Hope this helps!

**Comment by captainrandom** [comment]
Weird now I get the following when running `gf181 echo [] [3]`:

```
Decode Error: too few bytes
From:   demandInput


[]
```
and keelung seems to run fine with docker:
```
docker run -i btqag/keelungc --version
Keelung v0.9.0
```

**Comment by christam96** [comment]
@Zekt does this have something to do with the compiler? I think the latest version is `v0.9.3`?

**Comment by Zekt** [comment]
Yeah, usually `Decode Error: too few bytes` means the compiler version doesn't match, 
The latest examples run on v0.9.3, you can download the latest [binaries](https://github.com/btq-ag/keelung/releases/tag/v0.9.3-5), and I've updated the docker image as well, we are actively developing and this might happen from time to time so sorry in advance!
